bindings/java/src/main/java/org/openucx/jucx/ucp/UcpRequest.java

```diff
@@ -20,6 +20,9 @@
     private long recvSize;
 
     private UcpRequest(long nativeId) {
+        if (nativeId < 0) {
+            nativeId = 0L;
+        }
```

Comment:
maybe we should do this only is UCS_PTR_IS_PTR is true:
```
    jobject jucx_request = env->NewObject(jucx_request_cls, jucx_request_constructor,
                                          (native_ptr)request);
```
and if not true, initialize UcpRequest.native_id to null and keep it as null(maybe need another constructor), instead of initializing it to 0 and then later changing it to null